cli/cmd/dashboard.go

```diff
@@ -19,15 +23,33 @@ var dashboardCmd = &cobra.Command{
 	Short: "Open the Conduit dashboard in a web browser",
 	Long:  "Open the Conduit dashboard in a web browser.",
 	RunE: func(cmd *cobra.Command, args []string) error {
+		var dashboardCheckResult *healthcheckPb.CheckResult
 		if proxyPort <= 0 {
 			log.Fatalf("port must be positive, was %d", proxyPort)
 		}
 
-		kubectl, err := k8s.NewKubectl(shell.NewUnixShell())
+		sh := shell.NewUnixShell()
+		kubectl, err := k8s.NewKubectl(sh)
 		if err != nil {
 			log.Fatalf("Failed to start kubectl: %v", err)
 		}
 
+		kubeApi, err := k8s.NewK8sAPI(sh, kubeconfigPath)
+		if err != nil {
+			log.Fatalf("Failed to create Kubernetes API: %v", err)
```

Comment:
I think the text of this error message should be changed to "Failed to connect to the Kubernetes API") or something like that